web/src/components/datasets/DatasetDetailPage.tsx

```diff
@@ -127,6 +152,7 @@ const DatasetDetailPage: FunctionComponent<IProps> = (props) => {
       }}
     >
       <Box>
+        {formatTags(tags, tagData)}
```

Comment:
This isn't strictly needed as its been superseded by DatasetTags.

Reply:
So that is for the readonly view, but I can make a readonly mode inside that container.

Something like (it also lets you remove, which I guess is fine): 
![image](https://github.com/MarquezProject/marquez/assets/7514204/be9758c1-85f3-444b-a598-565c7c044a35)